ner/src/main/java/edu/illinois/cs/cogcomp/ner/ExpressiveFeatures/BrownClusters.java

```diff
@@ -88,7 +88,14 @@ public static void init(Vector<String> pathsToClusterFiles, Vector<Integer> thre
                 THashMap<String, String> h = new THashMap<>();
                 // We used to access the files as resources. Now we are accessing them programmatically.
                 // InFile in = new InFile(ResourceUtilities.loadResource(pathsToClusterFiles.elementAt(i)));
-                InputStream is = new FileInputStream(bcDirectory.getPath() + File.separator + pathsToClusterFiles.elementAt(i));
+                // Here we check if the file exists in data-store resources.
+                // If not, we interpret input path as a local resource.
+                String bcFilePath = bcDirectory.getPath() + File.separator + pathsToClusterFiles.elementAt(i);
+                File checker = new File(bcFilePath);
+                if (!checker.exists()){
+                    bcFilePath = pathsToClusterFiles.elementAt(i);
+                }
+                InputStream is = new FileInputStream(bcFilePath);
```

Comment:
@Slash0BZ this can could confusions. (for people thay may think that it is reading from datastore, but it is in fact reading locally). 
How about your add a switch to the configurator (say `boolean readFromDataStore`, with default value of `true`) and use it here? (rather than checking datastore to see if the file exists or not).

Reply:
Ok that makes more sense.